extensions/gmsa-dc/v1/Promote-DC.ps1

```diff
@@ -0,0 +1,67 @@
+<#
+ .SYNOPSIS
+ This script will automate the creation of a Forest, Domain, and Domain Controller
+
+ .NOTES
+ This is only for automated e2e testing.  DO NOT use this for production.
+ Jeremy Wood (JeremyWx)
+ Version: 1.0.0.0
+#>
+
+Set-StrictMode -Version Latest
+$ErrorActionPreference = 'Stop'
+$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
+
+#Set script variables - These will need changed if this extension is moved to a new repo!!!
+$GMSARoot = "C:\gmsa"
+#$ScriptURL = "https://raw.githubusercontent.com/kubernetes-sigs/windows-testing/master/extensions/gmsa-dc/v1"
+$ScriptURL = "https://raw.githubusercontent.com/JeremyWx/windows-testing/master/extensions/gmsa-dc/v1/"
```

Comment:
I think this has been left behind from testing. The proper URL is the one from L17